pkg/apiserver/filters/requestinfo.go

```diff
@@ -75,8 +75,9 @@ func WithRequestInfo(handler http.Handler, resolver request.RequestInfoResolver)
 }
 
 func injectToken(req *http.Request) (requestWithTokenContext *http.Request) {
-	authorization := req.Header.Get("Authorization")
+	authorization := req.Header.Get("X-Authorization")
 	token := strings.ReplaceAll(authorization, "Bearer ", "")
+	token = strings.ReplaceAll(token, "bearer ", "")
```

Comment:
Word `bearer` should be capital here.

Reply:
See also line 79.

`token := strings.ReplaceAll(authorization, "Bearer ", "")`

I keep using `bearer` and `Bearer` for compatibility purposes.

Reply:
My bad.

Reply:
No worries.